docs/flow-visibility.md

```diff
@@ -0,0 +1,45 @@
+#Flow Visibility in Antrea
+
+[Antrea](architecture.md) implements Container Network Interface(CNI) and Network Policy (both Kubernetes and customized)
+ to provide network connectivity and security features for pod workloads. Considering the enormity and dynamism of Kubernetes
+ workloads in a cluster, Flow Visibility helps in the management and configuration of Kubernetes constructs such as Network
+ Policy, Services, Pods etc., and there by enhance the performance and security aspects of pod workloads.
+ 
+##Flow Exporter feature
+
+ In Antrea, basic building block to realize network flow visibility is the **Flow Exporter feature**. Flow exporter operates
```

Comment:
you sometimes capitalize "Flow Visibility" and sometimes you don't
same comment for "Flow Exporter"

Reply:
Fixed this. Some times I wanted to use flow visibility not as a noun. Changed the wording in those cases.